libcxx/utils/ci/buildkite-pipeline.yml

```diff
@@ -209,8 +209,8 @@ steps:
   - label: FreeBSD 13 amd64
     command: libcxx/utils/ci/run-buildbot generic-cxx23
```

Comment:
I don't know. 

How about `https://github.com/H-G-Hristov/llvm-project/blob/c05baf241cdf8ca34c3a6f0182487d40eeca8490/libcxx/utils/ci/run-buildbot#L318-L319`:
`generic-cxx23` -> `generic-cxx26`?
@mordante is that the correct change?